app/helpers/reminder_helper.rb

```diff
@@ -4,22 +4,31 @@ module ReminderHelper
   def print_reminders(data)
     reminders = ''
 
-    if data.active_reminders&.any?
-      reminders = format_reminders(data.active_reminders.map(&:name))
+    if reminders?(data)
+      names_arr = data.active_reminders.map(&:name)
+      reminders = format_reminders(names_arr)
     end
 
     reminders.html_safe
   end
 
   private
 
+  def reminders?(data)
+    data.methods.include?(:active_reminders) && data.active_reminders&.any?
```

Comment:
Could you explain why you're also checking for data.methods to include active_reminders?

Reply:
Just a sanity check so this method does not have to rely on someone passing the correct data into it. If the data in this case doesnt have a method `active_reminders` the safe nav `&` didnt catch it. This could be caught in other specs but since I wasnt sure I opted to over test it here